share/models/meta.py

```diff
@@ -1,17 +1,41 @@
+import furl
+import json
+
 from django.db import models
 from django.db import IntegrityError
 from django.contrib.postgres.fields import JSONField
 
 from share.models.base import ShareObject
 from share.models.fields import ShareForeignKey, ShareURLField, ShareManyToManyField
-from share.apps import ShareConfig as share_config
 
 
-__all__ = ('Venue', 'Award', 'Tag', 'Link', 'Subject')
+__all__ = ('Venue', 'Award', 'Tag', 'Subject', 'Identifier', 'Relation')
 
 # TODO Rename this file
 
 
+class Identifier(ShareObject):
+    # https://twitter.com/berniethoughts/
+    url = ShareURLField(unique=True)
+
+    # https://twitter.com/
+    def get_base_url(self):
```

Comment:
I removed it since it was always inferable from `url`, but I can see how it'd be useful to store it separately. If we call it `domain` it shouldn't include the scheme, though...